Gruntfile.coffee

```diff
@@ -147,6 +147,30 @@ module.exports = ->
             pattern: /\$NOFLO_OAUTH_CLIENT_SECRET/ig
             replacement: process.env.NOFLO_OAUTH_CLIENT_SECRET or ''
           ,
+            pattern: /\$NOFLO_OAUTH_SSL_CLIENT_ID/ig
+            replacement: process.env.NOFLO_OAUTH_SSL_CLIENT_ID or ''
+          ,
+            pattern: /\$NOFLO_OAUTH_SSL_CLIENT_REDIRECT/ig
```

Comment:
Why do we need separate settings for SSL? What does it mean if these values are empty (the default)?